src/tests/authentication/test_token.py

```diff
@@ -0,0 +1,221 @@
+import json
+import uuid
+from datetime import datetime, timedelta
+from time import sleep
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+from django.core.exceptions import ObjectDoesNotExist
+from django.test import TestCase
+
+from chaosinventory.authentication.models import Token, User
+
+
+def get_db_token(key) -> Token:
+    return Token.objects.get(key=key)
+
+
+def create_user(**kwargs) -> User:
+    password = None
+    if 'password' in kwargs:
+        password = kwargs.pop('password')
+
+    default_user_data = {
+        'username': uuid.uuid4(),
+        'first_name': 'Demo',
+        'last_name': 'User',
+        'email': 'user@example.org',
+    }
+    user: User = get_user_model().objects.create(**{**default_user_data, **kwargs})
+
+    if password is not None:
+        user.set_password(password)
+        user.save()
+
+    return user
+
+
+class TokenTestCase(TestCase):
+    def setUp(self) -> None:
```

Comment:
This method should be named `set_up`.

Reply:
This comes down from the python unittest module....
https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp